pkg/remote/aws/sqs_queue_details_fetcher.go

```diff
@@ -0,0 +1,43 @@
+package aws
+
+import (
+	"strings"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+	"github.com/cloudskiff/driftctl/pkg/terraform"
+	"github.com/sirupsen/logrus"
+)
+
+type SQSQueueDetailsFetcher struct {
+	reader       terraform.ResourceReader
+	deserializer *resource.Deserializer
+}
+
+func NewSQSQueueDetailsFetcher(provider terraform.ResourceReader, deserializer *resource.Deserializer) *SQSQueueDetailsFetcher {
+	return &SQSQueueDetailsFetcher{
+		reader:       provider,
+		deserializer: deserializer,
+	}
+}
+
+func (r *SQSQueueDetailsFetcher) ReadDetails(res resource.Resource) (resource.Resource, error) {
+	ctyVal, err := r.reader.ReadResource(terraform.ReadResourceArgs{
+		ID: res.TerraformId(),
+		Ty: aws.AwsSqsQueueResourceType,
+	})
+	if err != nil {
+		if strings.Contains(err.Error(), "NonExistentQueue") {
+			logrus.Debugf("Ignoring %+v", err)
```

Comment:
Maybe a more explicit message with additional context could be more usefull in debug traces. We did it previously for EC2 instances

```go
logrus.WithFields(logrus.Fields{
			"id": res.TerraformId(),
		}).Debug("Instance read returned nil (instance may be terminated), ignoring ...")
```